test/proxy/transparent/TransparentUpgradeableProxy.behaviour.js

```diff
@@ -17,8 +19,21 @@ const InitializableMock = artifacts.require('InitializableMock');
 const DummyImplementation = artifacts.require('DummyImplementation');
 const ClashingImplementation = artifacts.require('ClashingImplementation');
 
-module.exports = function shouldBehaveLikeTransparentUpgradeableProxy(createProxy, accounts) {
-  const [proxyAdminAddress, proxyAdminOwner, anotherAccount] = accounts;
+module.exports = function shouldBehaveLikeTransparentUpgradeableProxy(createProxy, initialOwner, accounts) {
+  const [proxyCreator, anotherAccount] = accounts;
+
+  async function createProxyWithImpersonatedProxyAdmin(logic, initData, opts) {
+    const proxy = await createProxy(logic, initData, { from: proxyCreator, ...opts });
```

Comment:
the "from" part is not necessary, is it ?
```suggestion
    const proxy = await createProxy(logic, initData, opts);
```

Reply:
It's necessary because passing an empty object will throw as "unexpected 3 arguments, 2 are required"

Reply:
> It's necessary because passing an empty object will throw as "unexpected 3 arguments, 2 are required"

The issue is that if you call a function with an "empty" opts ({}) then web3 will not understand that as an opts, but as an argument. This is strange. 

The fix is to not forward the opts if its "null/undefined" ... and not assign it a default value.

This allows use to remove a big number of {from:...} that were not needed for anything but to make web3 understand that opts is an option object.

(see my commit [a22e182](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4382/commits/a22e182dadc491f00611493f764439045ef42c78))

Reply:
Yes, it was very strange when I first found it. I decided to ignore it focusing on make the tests work, thanks for pushing a fix, it also looks good to me!